integrations/google_ai/src/haystack_integrations/components/generators/google_ai/chat/gemini.py

```diff
@@ -317,6 +317,15 @@ def _get_response(self, response_body: GenerateContentResponse) -> List[ChatMess
             candidate_metadata = metadata["candidates"][idx]
             candidate_metadata.pop("content", None)  # we remove content from the metadata
 
+            # align openai api response
+            usage_metadata = metadata.get("usage_metadata")
+            if usage_metadata:
+                candidate_metadata["usage"] = {
+                    "prompt_tokens": usage_metadata["prompt_token_count"],
+                    "completion_tokens": usage_metadata["candidates_token_count"],
+                    "total_tokens": usage_metadata["total_token_count"],
+                }
```

Comment:
- my impression is that the key names are wrong. See https://ai.google.dev/api/generate-content#UsageMetadata
- if we want to add this information to each candidate, we should
  - use `prompt_tokens` as you did
  - use `completion_tokens` from `tokenCount` of each candidate (https://ai.google.dev/api/generate-content#v1beta.Candidate)
  - compute `total_tokens` as the sum of the previous to field.